2017/Team-Training/OMERO-apps/index.html

```diff
@@ -0,0 +1,1603 @@
+        <script>
+            document.write( '<link rel="stylesheet" href="reveal-js/css/print/' + ( window.location.search.match( /print-pdf/gi ) ? 'pdf' : 'paper' ) + '.css" type="text/css" media="print">' );
+        </script>
+    </head>
+
+    <body class="ome_theme">
+        <div class="reveal">
+            <!-- Any section element inside of this container is displayed as a slide -->
+            <div class="slides">
+
+                <!-- talk outline -->
+                <section>
+                    <h2>Web Apps Review</h2>
+                    <br/>
+                    <ul>
+                        <li>Environment/Setup</li>
+                        <li>'Best Practices/Design'</li>
+                        <li>Open With</li>
+                        <li>Figure vs. IViewer</li>
+                        <li>Example Figure</li>
+                        <li>Example IViewer</li>
+                        <li>IViewer Details</li>
+                        <li>Q & A</li>
+                    </ul>
+                </section>
+
+                <!-- app environment -->
+                <section>
+                    <h3>Environment</h3>
+                    <img src="images/app_environment.png"/>
+                </section>
+
+                <!-- app setup - dirs -->
+                <section>
+                    <h3>'Minimal' Setup – directories</h3>
+                    <br/>
+                    <ul>
+                        <li>app/static/app/app.js</li>
+                        <li>app/templates/app/index.html</li>
+                    </ul>
+                    <br/></br/>
+                    <img style="width: 500px" src="images/figure_dirs.png"/>
+                </section>
+
+                <!-- app setup - static/app -->
+                <section>
+                    <h3><i>static/app</i> – contents?</h3>
+                    <br/>
+                    <div style="float: left;">
+                        <br/>
+                        <ul>
+                            <li>JS</li>
+                            <li>CSS</li>
+                            <li>Images</li>
+                            <li>Fonts</li>
+                            <li>Third Party Resources</li>
+                        </ul>
+                    </div>
+                    <div style="float: right;">
+                        <img style="width: 500px" src="images/iviewer_dirs.png"/>
+                    </div>
+                </section>
+
+                <!-- app setup - static/templates/index.html -->
+                <section>
+                    <h2><i>app/templates/app/index.html</i></h2>
+                    <br/>
+                    <pre style="box-shadow: none;">
+                        <code style="max-height: 450px" class="xml">
+&lt;html&gt;
+    &lt;head&gt;
+      &lt;link rel="stylesheet" type="text/css"
+              href=" {% static 'app_directoryr/css/app.css?version=1.1' %}" /&gt;
```

Comment:
The 'r' in 'app_directoryr/css/...' here looks to be a typo

Reply:
yes, thank you